sdk/provisioning/Azure.Provisioning.CloudMachine/src/CloudMachineInfrastructure.cs

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Provisioning.Authorization;
+using Azure.Provisioning.EventGrid;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Resources;
+using Azure.Provisioning.Roles;
+using Azure.Provisioning.ServiceBus;
+using Azure.Provisioning.Storage;
+
+namespace Azure.Provisioning.CloudMachine;
```

Comment:
I wonder if we don't want to put it in Azure.CloudMachine namespace. Otherwise users have to import both namespaces in the hello world scenario. But, we can do the move later.

Reply:
Since provisioning and development can be separate, maybe this is OK. But we can decide later for sure.